src/Host/Client/Impl/rtvs/R/eval.R

```diff
@@ -67,6 +67,17 @@ describe_object <- function(obj, res, fields, repr = NULL) {
       res$dim <- as.list(dim);
     }
   }
+  
+  if (field('to_df')) {
+    can_coerce_to_df <- function(obj){
+      df_test <- function(objclass){
+        res <- getS3method('as.data.frame', objclass, optional = TRUE);
+        if (is.null(res)) { FALSE } else { TRUE }
+      }
+      any(as.logical(lapply(class(obj), df_test)), na.rm = TRUE);
+    }
+    res$to_df <- FALSE_if_error(can_coerce_to_df(obj));
```

Comment:
For a more compact representation, use `NULL` here if there is an error, and then on the other side, when parsing JSON, assume that missing field represents the default value of false. I believe other boolean flags here use this approach.